developer-docs-site/docs/cli-tools/aptos-cli-tool/use-aptos-cli.md

```diff
@@ -867,7 +867,47 @@ FAILURE proving 1 modules from package `hello_prover` in 0.067s
 ```
 In this case, see [Install the dependencies of Move Prover](install-aptos-cli#step-3-optional-install-the-dependencies-of-move-prover).
 
-### Debug and print stack trace
+### Profiling gas use
+
+This *experimental* feature lets you [profile gas usage](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/aptos-gas-profiling) in the Aptos virtual machine locally rather than [simulating transactions](../../concepts/gas-txn-fee.md#estimating-the-gas-units-via-simulation) at the [fullnode](https://fullnode.devnet.aptoslabs.com/v1/spec#/operations/simulate_transaction). You may also use it to visualize the transaction gas log, in the form of a flame graph.
```

Comment:
I feel like we don't necessarily have to mention the concept of the transaction gas log to the user as it is an internal implementation detail that is subject to change.